test/controllers/users_controller_test.rb

```diff
@@ -2,7 +2,13 @@
 
 class UsersControllerTest < ActionDispatch::IntegrationTest
   setup do
-    @user = users(:one)
+    @user = User.new
```

Comment:
Since you never call `@user.save` here, this user doesn't exist as far as the show/edit/update tests below are concerned. But I don't think the changes in this file are necessary since tests pass for me without them.